README.md

```diff
@@ -1,26 +1,48 @@
+| `limesurvey_admin_full_name`                       |       `Your Name`       | Full name of the administrator                                 |
+| `limesurvey_admin_email`                           | `your.name@example.com` | Admin email adress                                             |
+| `limesurvey_mail_admin_name`                       |       `Your Name`       | Sender name for outgoing emails                                |
+| `limesurvey_mail_admin_adress`                     | `your.name@example.com` | Sender adress of outgoing emails                               |
+| `limesurvey_mail_bounce_adress`                    | `your.name@example.com` | Email Adress for bounces                                       |
+| `limesurvey_ldap_plugin_server_url`                |   :heavy_check_mark:    | Url to your ldap server                                                 |
+| `limesurvey_ldap_plugin_server_port`               |          `389`          | Ldap port                                                 |
+| `limesurvey_ldap_plugin_ldap_version`              |   :heavy_check_mark:    | Ldap version either `2` or `3`'                                                 |
+| `limesurvey_ldap_plugin_opt_referrals`             |           `0`           | Select true if referrals must be followed (use false for ActiveDirectory)                                                |
+| `limesurvey_ldap_plugin_tls`                       |           `0`           | Use tls either `0` or `1`                                                |
+| `limesurvey_ldap_plugin_ldap_mode`                 |   :heavy_check_mark:    | How limesurvey should connect to ldap either `simplebind` or `searchandbind`                                                 |
+| `limesurvey_ldap_plugin_search_user_attribute`     |   :heavy_check_mark:    | Attribute to compare to the given login can be uid, cn, mail, ...                                                |
+| `limesurvey_ldap_plugin_user_search_base`          |   :heavy_check_mark:    | Base DN for the user search operation. Multiple bases may be separated by a semicolon (;)                                                 |
+| `limesurvey_ldap_plugin_binddn`                    |   ` `   |Optional DN of the LDAP account used to search for the end-user's DN. An anonymous bind is performed if empty.                                                 |
+| `limesurvey_ldap_plugin_bindpwd`                   |   ``    | Password of the LDAP account used to search for the end-user's DN if previoulsy set                                                 |
+| `limesurvey_ldap_plugin_mail_attribute`            |   :heavy_check_mark:    | LDAP attribute of email address                                                |
+| `limesurvey_ldap_plugin_full_name_attribute`       |   :heavy_check_mark:    | LDAP attribute of full name                                              |
```

Comment:
This is like the third time a full name is being used. Is it not normally used for the same purpose? If so, can you change this to one variable?

Reply:
no its not for the same purpose one time its for all emails on for error emails and this is an attribute to load a name from the specified ldap attribute